### DISCUSSION TITLE
Declarations when tables have same name

### DISCUSSION THREAD

**Discussion by bigrogerio** [dataform-co/dataform#1470] [Q&A] [answered]
hi! i'm heavily using Dataform in my work at BigQuery in Google Cloud Platform. Dataform really changed the game in my job.

I have a question that I didn't find the answer in official Docs:

Suppose I have two tables **with the same name** from different schemas but the same database.  Even if I name each declaration .sqlx file differently, how can I invoke these tables through ${ref()} statement if the tables share the same name?

thank you very much for all the attention.

best regards.

**Comment by BenBirt** [comment, marked as answer]
`ref("database", "schema", "name")`, or if the schema name is enough to disambiguate, then just `ref("schema", "name")` will work.

**Comment by bigrogerio** [comment]
![image](https://user-images.githubusercontent.com/62626374/227531274-954bad57-21ac-4956-a7bc-eac3239aad10.png)

For further reference.
